shared/engine/index.js

```diff
@@ -306,6 +306,7 @@ const makeEngine = () => {
 
   if (!engine) {
     engine = (process.env.KEYBASE_NO_ENGINE || isTesting) ? new FakeEngine() : new Engine()
+    console.log('aaa', engine)
```

Comment:
i'll remove this was just me trying to debug why i was getting duplicate engines